### ISSUE TITLE
It is not clear to me how to use the package in laravel 5.5

### ISSUE THREAD

**Issue by cesarlanz** [gocanto/places-validation#20]
Greetings, it is not clear to me how to use the package in laravel 5.5. It does not work for me, can you help me?

**Comment by gocanto** [comment]
you can use it as a normal validation rule.

```php
$this->validate($request->all(), [
   'adress' => 'required|valid_place'
])
```

**Comment by cesarlanz** [comment]
I did it but, it does not generate any error when entering invalid addresses.

```
    public function updateAddress(Request $request)
    {
        $request->validate([
            'google_address' => 'required|string|valid_place',
            'street'    => 'nullable|string',
            'address'   => 'nullable|string',
            'city'      => 'required|string',
            'state'     => 'required|string',
            'country'   => 'required|string',
            'zip'       => 'nullable|string',
        ]);

        $account = \Auth::user()->account;
        $account->fill($request->all());
        $account->save();

        return response()->json($account);
    }
```

**Comment by gocanto** [comment]
umm.. there might be an issue then. 

Here are the tests I have for it https://github.com/gocanto/places-validation/blob/master/tests/ManagerTest.php

you can check them out and fix something if broken?
